bindings/java/build-windows.xml

```diff
@@ -30,6 +30,14 @@
 	</target>
 
 	<target name="check-native-build" depends="check-build-32,check-build-64"/>
+	<target name="set-library-path" description="set-library-path">
```

Comment:
We now have 3 copies of this code. We should redesign this so that all builds will get the libraries out of 'lib' and only offline will go to the other location.